EventFiltering/PWGCF/CFFilterAll.cxx

```diff
@@ -1097,18 +1209,20 @@ struct CFFilter {
   template <typename T>
   bool isSelectedTrackPID(T const& track, CFTrigger::ParticleSpecies partSpecies, bool Rejection, double nSigmaTPC[2], int charge)
   {
-    // nSigma should have entries [proton, deuteron]
+    // nSigma should have entries [proton, deuteron, pion]
     bool isSelected = false;
     bool pThres = true;
     float nSigma = -999.;
-    float nSigmaIts = -999.;
+    float nSigmaIts = 0.;
```

Comment:
Why did you change the default value? The default value should be outside the selection range.

Reply:
Changed on the request of Anton, change is reverted and some thresholds for the pions adjusted.